tests/parsers/presets.py

```diff
@@ -11,42 +11,78 @@
 from tests import test_lib as shared_test_lib
 
 
-class ParserPresetDefinitionTest(shared_test_lib.BaseTestCase):
-  """Tests for the parser and parser pluging preset definition."""
+class ParserPresetTest(shared_test_lib.BaseTestCase):
+  """Tests for the parser and parser plugin preset."""
 
   def testInitialize(self):
     """Tests the __init__ function."""
-    test_definition = presets.ParserPresetDefinition()
+    test_definition = presets.ParserPreset('test', ['parser1', 'parser2'])
     self.assertIsNotNone(test_definition)
 
 
-class ParserPresetsTest(shared_test_lib.BaseTestCase):
-  """Tests for the parser and parser pluging presets."""
+@shared_test_lib.skipUnlessHasTestFile(['presets.yaml'])
+class ParserPresetsManagerTest(shared_test_lib.BaseTestCase):
+  """Tests for the parser and parser plugin presets manager."""
 
   def testGetNames(self):
     """Tests the GetNames function."""
```

Comment:
Please add a test that checks that the names are in alphabetical order, per the GetNames docstring.

Reply:
ack